### PR TITLE
Update the Pizzas example

### PR DESCRIPTION
**Pull request by mmitoraj**
**Description**

Changes proposed in this pull request:

- Update the Pizzas example

**Related issue(s)**

### CODE REVIEW

File: examples/pizzas/README.md [hunk 1]
```diff
@@ -1,27 +1,41 @@
...
+   kubectl apply -f samples/ --recursive
+   ```
+
+## Result
+
+After uploading the files, refresh the browser. You should see the new items in the left-side navigation.
+
+On the cluster level, in the **Extensions** section, you can find the two applied ConfigMaps, `pizzaorders` and `pizzas`. You can see the configuration details in the General, Form Fields, List Columns, Details View, Data Sources, or Translation sections taken from the `configuration` directory.
+
+Go to **Namespaces** to find the newly created `pizzas` namespace. Go into the namespace and in the left-side navigation, in the **Lunch** category, see the details of two UI components: **Pizza Orders** and **Pizzas** with all the details taken from the `samples` directory.
```
Comment (nataliasitko) [inline examples/pizzas/README.md:41]:
```suggestion
Go to **Namespaces** and select the newly created `pizzas` namespace. In the **Lunch** category of the left-side navigation, see two UI components: **Pizza Orders** and **Pizzas**. All the details of these components are taken from the `samples` directory.
```

File: examples/pizzas/README.md [hunk 2]
```diff
@@ -1,27 +1,41 @@
...
-The `configuration` directory consists of two CustomResourceDefinitions (CRD) and two ConfigMaps that store the information about the extensibility configuration.
+The `configuration` directory consists of two CustomResourceDefinitions (CRDs) and two ConfigMaps that store the information about the extensibility configuration.
 
-The `samples.yaml` file includes the examples of the above CRDs.
+The `samples` directory includes YAML files with examples of the above CRDs.
 
 ## Installation
 
-To test the extensibility feature using the Pizza example, simply upload all the YAML files from the `configuration` directory and the `samples.yaml` file to your cluster or use the following commands:
+To test the extensibility feature using the Pizzas example, upload all the YAML files from the `configuration` and `samples` directories to your cluster. You can use the following commands:
```
Comment (nataliasitko) [inline examples/pizzas/README.md:13]:
```suggestion
To test the extensibility feature using the Pizzas example, upload all the YAML files from the `configuration` and `samples` directories to your cluster. Follow the steps:
```

File: examples/pizzas/README.md [hunk 3]
```diff
@@ -1,27 +1,41 @@
-# Pizza
+# Pizzas
 
 ## Overview
 
 The `pizzas` example showcases the Busola's extensibility feature.
 
-The `configuration` directory consists of two CustomResourceDefinitions (CRD) and two ConfigMaps that store the information about the extensibility configuration.
+The `configuration` directory consists of two CustomResourceDefinitions (CRDs) and two ConfigMaps that store the information about the extensibility configuration.
```
[the remark this answers is not in the document]
Comment (mmitoraj) [reply examples/pizzas/README.md:7]: or 
```
The [`configuration`](https://github.com/kyma-project/busola/tree/main/examples/pizzas/configuration) directory consists of two CustomResourceDefinitions (CRDs) and two ConfigMaps that store the information about the extensibility configuration.
```

File: examples/pizzas/README.md [hunk 4]
```diff
@@ -1,27 +1,41 @@
...
 
-The `samples.yaml` file includes the examples of the above CRDs.
+The `samples` directory includes YAML files with examples of the above CRDs.
```
[the remark this answers is not in the document]
Comment (mmitoraj) [reply examples/pizzas/README.md:9]: or 

```
The [`samples`](https://github.com/kyma-project/busola/tree/main/examples/pizzas/configuration) directory includes YAML files with examples of the above CRDs.
```

File: examples/pizzas/README.md [hunk 5]
```diff
@@ -1,27 +1,41 @@
...
 ## Overview
 
-The `pizzas` example showcases the Busola's extensibility feature.
+The `pizzas` example showcases Busola's extensibility feature.
 
-The `configuration` directory consists of two CustomResourceDefinitions (CRD) and two ConfigMaps that store the information about the extensibility configuration.
+The [`configuration`](https://github.com/kyma-project/busola/tree/main/examples/pizzas/configuration) directory consists of two CustomResourceDefinitions (CRDs) and two ConfigMaps that store the information about the extensibility configuration.
 
-The `samples.yaml` file includes the examples of the above CRDs.
+The [`samples`](https://github.com/kyma-project/busola/tree/main/examples/pizzas/configuration) directory includes YAML files with examples of the above CRDs.
```
Comment (nataliasitko) [inline examples/pizzas/README.md:9]:
```suggestion
The [`samples`](https://github.com/kyma-project/busola/tree/main/examples/pizzas/samples) directory includes YAML files with examples of the above CRDs.
```

**Review by nataliasitko — APPROVED** [review]

**Review by akucharska — APPROVED** [review]

**Pull request merged**